preferencesfx/src/main/java/com/dlsc/preferencesfx/model/PreferencesFxModel.java

```diff
@@ -304,10 +298,20 @@ public void fireEvent(PreferencesFxEvent event) {
     if (list == null) {
       return;
     }
-    for (EventHandler<? super PreferencesFxEvent> eventHandler: list) {
+    for (EventHandler<? super PreferencesFxEvent> eventHandler : list) {
       if (!event.isConsumed()) {
         eventHandler.handle(event);
       }
     }
   }
+
+  /**
+   * Saves the settings, when {@link #isSaveSettings()} returns {@code true}.
+   */
+  public void saveSettings() {
```

Comment:
could you please also replace the individual calls in the method from `PreferencesFxDialog` in the method `private void setupDialogClose() {` with a call to this?